js/Source/Slider.Modify/Behavior.Slider.Modify.js

```diff
@@ -61,6 +64,13 @@ Behavior.addGlobalFilter('Slider.Modify', {
         roundAfterSnap: api.getAs(Number, 'roundAfterSnap')
       }
     );
+    if(formTarget){
+      var timer;
+      slider.addEvent('move', function(){
+        clearTimeout(timer);
+        timer = setTimeout(formTarget.submit, api.get('formSubmitDelay'));
+      });
+    }
```

Comment:
Examples you can find in our codebase:
* Behavior.InteractiveMessageList
* Behavior.BS.FormValidator
* Behavior.BS.Popup.FormRequest

Note that the `api` object the plugin is handed is *for the plugin* so if you have a behavior called `Foo` and a plugin called `Foo.Bar` the api instance handed to the plugin knows how to read `data-foo-bar-options`. If you want to read options from the `foo` portion of the element, you have to create your own API instance, which is easy:

`var formRequestAPI = new BehaviorAPI(element, 'foo');`

You can see an example of this in `Behavior.BS.Popup.FormRequest`